Translation/en/Text/Mods/Maker/ao_face.txt

```diff
@@ -187,8 +187,8 @@ M字バングストレート(右)=M-Shaped Straight Bangs (R)
 ふわカール B=Fluffy Curled B
 ふわカール C=Fluffy Curled C
 ふわカール D=Fluffy Curled D
-斜めぱっつん(右)=Slanting Straight-Cut Bangs (R)
```

Comment:
Hmm, we standardized on `(L)`/`(R)` in KKS. Is there a ported regex that's reverting these?

Reply:
Fixed